### PR TITLE
Lots of csv

### PR DESCRIPTION
**Pull request by SpookyBeverage**

### CODE REVIEW

File: apps/backend/app.py [hunk 1]
```diff
@@ -49,17 +50,28 @@ def run_batch(data):
...
     #Downloading Experiment File
     os.makedirs(f'ExperimentFiles/{id}')
     os.chdir(f'ExperimentFiles/{id}')
+    if experimentOutput != '':
+        print('There will be experiment outputs')
+        os.makedirs('ResCsvs')
     print(f'Downloading file for {id}')
-    filepath = experiment['file']
+    try:
+        filepath = experiment['filee']
```
Comment (budak7273) [inline apps/backend/app.py:64]: What is a `filee`?

File: apps/backend/app.py [hunk 2]
```diff
@@ -156,21 +190,13 @@ def frange(start, stop, step=None):
...
-        if step == "":
-            if(otherVar['type'] == 'integer'):
-                step = 1
-            elif(otherVar['type'] == 'float'):
-                step = 0.1
-    except KeyError:
-        print('OtherVar did not have type attribute')
-
     if otherVar['type'] == 'integer':
+        step = 1 if otherVar['step'] == '' else otherVar['step']
```
Comment (budak7273) [inline apps/backend/app.py:194]: python ternary statement? 😳

Should probably extract the default steps to be named constants

File: apps/frontend/components/NewExp.js [hunk 3]
```diff
@@ -126,7 +136,9 @@ const InformationStep = ({ form, ...props }) => {
 					>
 						<div
 							className='h-full grow-0 max-h-fit mb-4 overflow-y-scroll p-4 border-2 border-gray-300 border-dashed rounded-lg hover:border-gray-400'
-							style={{ maxHeight: '32vh' }}
+							// style={{ maxHeight: '32vh' }}
```
Comment (budak7273) [inline apps/frontend/components/NewExp.js:139]:
```suggestion
```

File: apps/frontend/pages/dashboard.js [hunk 4]
```diff
@@ -193,6 +193,13 @@ const ExpLog = ({projectinit}) => {
 						Download Results
 				</button> 
 				: ' '}
+				{project['finished'] == true && project['fileOutput'] != '' ?
```
Comment (budak7273) [inline apps/frontend/pages/dashboard.js:196]: are you sure this is the behavior you want for undefined?
image

**Review by budak7273 — CHANGES_REQUESTED** [review]

File: apps/frontend/pages/dashboard.js (hunk 4 above)
Comment (SpookyBeverage) [reply apps/frontend/pages/dashboard.js:196]: if fileOutput is undefined then something's wrong the default for fileOutput is the empty string so it should always be  either an output location or the empty string

File: apps/backend/app.py (hunk 2 above)
Comment (SpookyBeverage) [reply apps/backend/app.py:194]: Python Ternary Statement 😎, also will add named constants

File: apps/backend/app.py (hunk 1 above)
Comment (SpookyBeverage) [reply apps/backend/app.py:64]: leftover from testing, changed it back

**Review by budak7273 — APPROVED** [review]

**Pull request merged**